nuttx/include/stdint.h

```diff
@@ -55,80 +55,80 @@
 
 /* Limits of exact-width integer types */
 
-#define INT8_MIN            0x80
-#define INT8_MAX            0x7f
-#define UINT8_MAX           0xff
+#define INT8_MIN            (-INT8_MAX - 1)
+#define INT8_MAX            127
+#define UINT8_MAX           255
 
-#define INT16_MIN           0x8000
-#define INT16_MAX           0x7fff
-#define UINT16_MAX          0xffff
+#define INT16_MIN           (-INT16_MAX - 1)
+#define INT16_MAX           32767
+#define UINT16_MAX          65535u
 
 #ifdef __INT24_DEFINED
-#  define INT24_MIN         0x800000
-#  define INT24_MAX         0x7fffff
-#  define UINT24_MAX        0xffffff
+#  define INT24_MIN         (-INT24_MAX - 1)
+#  define INT24_MAX         8388607
+#  define UINT24_MAX        16777215u
 #endif
 
-#define INT32_MIN           0x80000000
-#define INT32_MAX           0x7fffffff
-#define UINT32_MAX          0xffffffff
+#define INT32_MIN           (-INT32_MAX - 1)
+#define INT32_MAX           2147483647
+#define UINT32_MAX          4294967295u
 
 #ifdef __INT64_DEFINED
-#  define INT64_MIN         0x8000000000000000
-#  define INT64_MAX         0x7fffffffffffffff
-#  define UINT64_MAX        0xffffffffffffffff
+#  define INT64_MIN         (-INT64_MAX - 1ll)
```

Comment:
These are taken from upstream nuttx. I strongly advise against deviating from that. If we want capital letters would be good to do that on upstream.